internal/appconfig/watcher.go

```diff
@@ -493,25 +497,46 @@ func configVarSortFunc(vars []*pb.ConfigVar) func(i, j int) bool {
 	}
 }
 
+// Used to just in tracking from the config split, through eval, and back
+// to exporting.
```

Comment:
nitpick but would explain what static vs dynamic is here at a high level here and in the comment above `dynamicVar`

Reply:
also nitpick: confusing wording `Used to just in tracking`

Reply:
Both fixed!